c2cgeoportal/scaffolds/create/project.yaml.in_tmpl

```diff
@@ -1,6 +1,6 @@
 project_folder: {{project}}
 project_package: {{package}}
 host: <host>
-checker_path: /${instance-id}/wsgi/check_collector?
+checker_path: /${instanceid}/wsgi/check_collector?
```

Comment:
Shouldn't it be ``${vars:instanceid}`` as in https://github.com/camptocamp/c2cgeoportal/blob/1.5/c2cgeoportal/scaffolds/create/config.yaml.in_tmpl#L91